src/components/apikey/APIKeyCreatePage.tsx

```
@@ -0,0 +1,153 @@
+    setInProgress(true);
+
+    try {
+      const created = await k8sCreate<APIKeyKind>({
+        model: getModelFromResource(formData),
+        data: formData,
+      });
+
+      // Navigate to detail view
+      navigate(
+        `/k8s/ns/${created.metadata?.namespace}/apikeys/${created.metadata?.name}`
+      );
```

ESLint

[error] Replace `⏎········`/k8s/ns/${created.metadata?.namespace}/apikeys/${created.metadata?.name}`⏎······` with ``/k8s/ns/${created.metadata?.namespace}/apikeys/${created.metadata?.name}``

(prettier/prettier)